internal/mysql/queries.go

```diff
@@ -123,4 +125,6 @@ var DefaultQueries = map[string]string{
 	querySetInnodbFlushLogAtTrxCommit: `SET GLOBAL innodb_flush_log_at_trx_commit = :level`,
 	queryGetReplicationSettings:       `SELECT @@innodb_flush_log_at_trx_commit as InnodbFlushLogAtTrxCommit, @@sync_binlog as SyncBinlog`,
 	querySetSyncBinlog:                `SET GLOBAL sync_binlog = :sync_binlog`,
+	queryGetMdbReplMonTs: 				`SELECT UNIX_TIMESTAMP(ts) AS ts FROM mysql.mdb_repl_mon`,
```

Comment:
This one looks very specific to your installation. Probably, we should use data from `replication_applier_status_by_worker` table OR move `mdb_repl_mon` to mysync.